cl/search/migrations/0026_alter_docket_unique_together_and_more.sql

```diff
@@ -0,0 +1,9 @@
+--
+-- Alter unique_together for docket (0 constraint(s))
+--
+ALTER TABLE "search_docket" DROP CONSTRAINT "search_docket_docket_number_pacer_case_a3184727_uniq";
+--
+-- Alter field docket_number on docket
+--
+DROP INDEX CONCURRENTLY IF EXISTS "search_docket_docket_number_b2afb9d6";
+DROP INDEX CONCURRENTLY IF EXISTS "search_docket_docket_number_b2afb9d6_like";
```

Comment:
Django did a (dumb? necessary? annoying?) thing years ago and decided to change how indexes are named. As a result, when you create SQL files in a modern version of Django, it generates index names the way that version does, instead of the way the earlier version _did_. 

So if we ran this on our server, which was created with the old version, it wouldn't work because the index names are wrong. 

The correct place to get index names is from the DB itself. You could get these values from the dev DB, from our schema dump in the bulk data directory, or from the details of the issue I filed. :)

Reply:
Got it!. I'll update the SQL file.